src/charge_controller/dev/BMSManager.cpp

```diff
@@ -1,12 +1,138 @@
+
 #include <charge_controller/dev/BMSManager.hpp>
-int BMSManager::getBattVoltage() { return 0; }
 
-int BMSManager::getBattTemperature() { return 0; }
 
-bool BMSManager::isConnected() { return false; }
 
-bool BMSManager::faultDetected() { return false; }
+BMSManager::BMSManager(IO::CAN& can, IO::GPIO *bmsOK[]) {
+    this->can = &can;
+    for (int i = 0; i < MAX_BMS_PACKS; i++) {
+        this->bmsOK[i] = bmsOK[i];
+    }
+};
+
+//int BMSManager::getStateOfCharge() { return 0; }
+//
+//void BMSManager::updateBMSData(IO::CANMessage& message) {
+//    uint8_t length = message.getDataLength();
+//    LOG.log(Logger::DEBUG, "Got can message with id 0x%X and length %u", message.getId(),message.getDataLength());
+//    if(message.getId() == TOTAL_VOLTAGE_ID){
+//
+//        if(length == 4){
+//            millivolts = *((uint32_t*) message.getPayload());
+//            LOG.log(Logger::DEBUG,"Got voltage update %u", millivolts);
+//        }
+//    } else if(message.getId() == STATE_ID){
+//        if(length == 1){
+//            BMSStatus data = *((BMSStatus*) message.getPayload());
+//            LOG.log(Logger::DEBUG,"Got state update %u", data);
+//            BMSStatus state = data;
+//        }
+//    }
+//}
```

Comment:
Can this code be removed or have documentation explaining why it is commented out